frontend/test/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html lang="en-US">
+
+<head>
+<!------ Include the above in your HEAD tag ---------->
```

Comment:
comment refers to removed code, but this file will be deleted by the frontend devs anyway

Reply:
Not going resolve this